torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1460,6 +1483,100 @@ Error PyTorchModelLoader::loadDequantize(const torch::jit::Node *ptNode) {
   return addValueMapping(outputs[0], dn->getResult());
 }
 
+Error PyTorchModelLoader::loadQuantizedConvUnpacked(
+    const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 9, outputs, 1));
+
+  glow::NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      input,
+      getGlowNodeValueForValue(inputs[QuantizedUnpackedConv2dInputs::input]));
+  input = rescaleUIntToInt(input);
+
+  input = F_.createTranspose("qconv_input_transposed", input, NCHW2NHWC);
+  glow::ShapeNHWC inputShape(input.dims());
+
+  glow::NodeValue weights;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      weights,
+      getGlowNodeValueForValue(inputs[QuantizedUnpackedConv2dInputs::weights]));
+  weights = rescaleUIntToInt(weights);
+  weights = F_.createTranspose("qconv_weights_tranposed", weights, NCHW2NHWC);
+  glow::ShapeNHWC weightsShape(weights.dims());
+
+  glow::NodeValue bias;
+  if (hasGlowNodeValueForValue(inputs[QuantizedUnpackedConv2dInputs::bias])) {
+    ASSIGN_VALUE_OR_RETURN_ERR(
+        bias,
+        getGlowNodeValueForValue(inputs[QuantizedUnpackedConv2dInputs::bias]));
+  } else {
+    glow::Tensor biasT(glow::ElemKind::FloatTy, {weightsShape.n});
+    biasT.zero();
+    glow::Constant *biasConstant =
+        F_.getParent()->createConstant("qconv_bias", std::move(biasT));
+    bias = biasConstant->getOutput();
+  }
+  auto biasType =
+      F_.getParent()->uniqueType(glow::ElemKind::Int32QTy, bias.dims(), 1.0, 0);
```

Comment:
I think the bias scale needs to be input scale * weight scale. @rdzhabarov can you recall if that's correct?

Reply:
Wonder what is our conclusion?

Reply:
Let's try changing it, I think this is correct